js/components/report/interactive-iframe.js

```diff
@@ -18,6 +19,14 @@ export default class InteractiveIframe extends PureComponent {
       this.iframePhone.post("initInteractive", typeof state === "string" ? JSON.parse(state) : state);
     };
     this.iframePhone = new iframePhone.ParentEndpoint(this.refs.iframe, phoneAnswered);
+
+    this.iframePhone.addListener("getFirebaseJWT", (options) => {
+      fetchClassData()
+        .then(classData => fetchFirestoreJWT(classData.class_hash, options.firebase_app))
+        .then(json => this.iframePhone.post("firebaseJWT", json))
```

Comment:
These lines aren't covered by a test.  I guess this will be tricky since you'll need to mock the embedding. Or mock iframePhone.

Reply:
If the mock report data interactive makes this getFirebaseJWT request, then this would probably be more covered by the existing cypress test. For the call to succeed though, the cypress tests would need to mock the request that is triggered by fetchFirestoreJWT but that shouldn't be hard using cypress' network mocking.

Reply:
@dougmartin just a ping on this. Perhaps we should give up on it at this point, but if you've got a little more patience for this, it seems like doing this before the Log Manager stuff would be good.

Reply:
Will do today.